data-model.json

```diff
@@ -0,0 +1,149 @@
+{   
+    "name": "Name1 Name2 County",  
+    "update_date": "yyyy/mm/dd ",
+    "update_time": "hh:mm [am/pm]",
```

Comment:
FWIW, I think 24 hour time would be preferable here. Basically, treat this as the time component of an W3C-style ISO8601 datetime.

I also might have misunderstood from the Slack conversation, but I was thinking the issue with separating dates and times was just for each item in the timeseries, and for the whole-dataset-update-time, we could just have an actual datetime, e.g. `yyyy-mm-ddThh:mmz`. If I got that wrong, though, that’s ok. I’m not looking to open up a whole new can of worms here. :)

Reply:
I agree that 24h time is better here

Reply:
Yes, I agree with having a whole-dataset-update-time per county. Would this format: `yyyy-mm-ddThh:mmz`  be a string? Some of the datasets I've seen on arcgis are using timestamps (ms since 1970/1/1).

Reply:
I think either is ok, but I generally much prefer [the W3C take on ISO 8601](https://www.w3.org/TR/NOTE-datetime) (i.e. the format you just suggested, @elaguerta). It has 2 big benefits:
1. Somewhat human readable while still being easy to parse and sortable, and
2. No confusion over whether the timestamp is milliseconds (friendly to JS) or seconds (more common for most other libraries and *nix tools) since Unix epoch.

Reply:
(It can also carry time zone info, which is super valuable, even when dealing with data that’s always coming from one geographic area, since most servers are configured to use UTC.)

Reply:
I prefer ISO 8601 format as well. ex) "2020-04-26T08:00:00Z"

Reply:
So the final decision is that we're using `yyyy-mm-ddThh:mmz` correct?

Reply:
Yep, it seems like we’re all largely in agreement on that. 👍